exporter/stefexporter/exporter.go

```diff
@@ -88,7 +88,7 @@ func (s *stefExporter) Start(ctx context.Context, host component.Host) error {
 	s.connMan.Start()
 
 	// Wrap async implementation of sendMetricsAsync into a sync-callable API.
-	s.sync2Async = internal.NewSync2Async(s.set.Logger, s.cfg.QueueConfig.NumConsumers, s.sendMetricsAsync)
+	s.sync2Async = internal.NewSync2Async(s.set.Logger, s.cfg.QueueConfig.Get().NumConsumers, s.sendMetricsAsync)
```

Comment:
`Get()` will return nil if the config is unset, right? So does this need changing to prevent a nil dereference panic?